tools/python/find_optimizer_opset_version_updates_required.py

```diff
@@ -59,18 +63,52 @@ def get_call_args_from_file(filename, function_or_declaration):
     return results
 
 
+def get_multiline_call_args_from_file(filename: str, function_or_declaration: str) -> typing.List[str]:
+    """
+    Search a file for all function calls or declarations that match the provided name.
+    Allows the opening '(' and closing ')' to be split across multiple lines.
+    Supports a single call per line.
+    """
+
+    results = []
+    with open(filename) as f:
+        function_and_args = None
+
+        for line in f:
+            if not function_and_args:
+                # look for new match
+                start = line.find(function_or_declaration)
+                if start != -1:
+                    function_and_args = line[start:].strip()
+            else:
+                # append to existing line and look for closing ')'
+                start = len(function_and_args)
+                function_and_args += line.strip()
+
+            end = -1
```

Comment:
```suggestion
```

From lgtm: This assignment to 'end' is unnecessary as it is redefined [here](https://lgtm.com/projects/g/microsoft/onnxruntime/snapshot/4d78d57e72f731c4dd181c7f7a913897c65f41f6/files/tools/python/find_optimizer_opset_version_updates_required.py#x4b15fc8ebf7a2d07:1) before this value is used.